app/_src/gateway/reference/configuration/configuration-3.7.x.md

```diff
@@ -2716,19 +2723,16 @@ values:
 
 The SSL key for `admin_gui_listen` values with SSL enabled.
 
-values: 
-* absolute path to the certificate key 
-* certificate key content 
-* base64 encoded certificate key content
+values: * absolute path to the certificate key * certificate key content *
+base64 encoded certificate key content
 
 **Default:** none
 
 
 ### admin_gui_flags
 {:.badge .free}
 
-Alters the layout Admin GUI (JSON) The only supported value is `{
-"IMMUNITY_ENABLED": true }` to enable Kong Immunity in the Admin GUI.
+Alters the layout Admin GUI (JSON) to enable Kong Immunity in the Admin GUI.
```

Comment:
This is weird. This flag shouldn't exist, if all it does it enable Kong Immunity. Can you please file a bug report? Immunity was deprecated & removed from the product in 2.7: https://docs.konghq.com/gateway/changelog/#deprecated-2